# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1325868005 18000
#      Fri Jan 06 11:40:05 2012 -0500
# Branch stable
# Node ID d0dcf26c4cbd76d7e9343ee9bd5ec08e91efd6ad
# Parent  d5d3037cbc11686e18662742a89b8cedb46e93ea
remove scary warning from startup message

* version.h.in (OCTAVE_STARTUP_MESSAGE): Delete scary warning about
changed semantics for arithmetic operators.

diff --git a/src/version.h.in b/src/version.h.in
--- a/src/version.h.in
+++ b/src/version.h.in
@@ -82,13 +82,10 @@ For more information, visit http://www.o
   OCTAVE_BUGS_STATEMENT
 
 #define OCTAVE_NAME_VERSION_COPYRIGHT_COPYING_WARRANTY_AND_BUGS \
   X_OCTAVE_NAME_VERSION_COPYRIGHT_COPYING_WARRANTY_AND_BUGS ("")
 
 #define OCTAVE_STARTUP_MESSAGE \
   X_OCTAVE_NAME_VERSION_COPYRIGHT_COPYING_WARRANTY_AND_BUGS \
     ("  For details, type `warranty'.") "\n\n" \
-  "For information about changes from previous versions, type `news'.\n\n\
-** WARNING ** Semantics for arithmetic operators were changed\n\
-              in version 3.6.0 and now broadcast.\n\
-              Type `doc broadcast' to get a description of the changes."
+  "For information about changes from previous versions, type `news'."
 #endif
